CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+To make edits to this site, just click the edit icon on a page on the live site to launch the GitHub code editor.
+If you are a member of the lab you can commit straight to `master` and for 95% of changes that is what you should do;
+do not bother submitting pull requests for this repo; we can always revert if we need to.
+
+After committing to `master`, the site will rebuild and redeploy to https://neuro.polymtl.ca/ within a minute or so. 
+You can watch its progress on the [Actions Tab](https://github.com/neuropoly/neuro.polymtl.ca/actions/workflows/publish.yml)!
+
+## Copyright
+
+By submitting content, you agree to license it under the terms set out in [LICENSE](./LICENSE) -- the [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/) license,
+and that you have the right to submit it -- it's either your own work or copy-pasted from content that's also open (like answers on StackOverflow).
+
+To honour the "BY" clause, your contributions will be credited by the `git log`, which can be viewed [here](https://github.com/neuropoly/neuro.polymtl.ca/graphs/contributors).
```

Comment:
Is this good enough to be legally binding?

Reply:
Also is it clear and fair and accessible?

Reply:
I think it's clear, fair and accessible but if you want to make it even clearer and avoid any potential error we can add an info text on a Pull-Request template of Github

Reply:
Smooth idea. I added this: https://github.com/neuropoly/neuro.polymtl.ca/blob/cc-by-sa-4.0/.github/workflows/pull_request_template.md?plain=1